builtin/logical/nomad/path_creds_create.go

```diff
@@ -17,6 +17,12 @@ const maxTokenNameLength = 256
 func pathCredsCreate(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: "creds/" + framework.GenericNameRegex("name"),
+
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixNomad,
+			OperationSuffix: "credentials",
```

Comment:
might be semantics, but would stick with `creds` instead of the full word. it would be confusing if it didn't match the path

Reply:
The idea behind DisplayAttributes -> OperationID's is to give more human-readable names for the generated methods. The methods names will often deviate from the API path. I prefer not to abbreviate whenever possible to be inclusive of people who don't speak English natively.

Reply:
if there are plugin guidelines (how to write a vault plugin/plugin best practices/etc) not using abbreviations would be good to put there.
at least mention for the openapi spec parts of it

Reply:
Changed to this:
```
nomad-configure-access
nomad-read-access-config
nomad-delete-access-config
```